### PR TITLE
Add tile features documentation

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

CleanShot 2022-11-28 at 17 49 19

Add documentation for tile features.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/frontend/pull/14483
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_dashboards/tile.markdown [hunk 1]
```diff
@@ -49,6 +49,10 @@ icon_tap_action:
   required: false
   description: Action taken on icon card tap. See [action documentation](/dashboards/actions/#tap-action). By default, it will `toggle` the entity (if possible), otherwise, show the "more-info" dialog.
   type: map
+extras:
+  required: true
```
Comment (ludeeus) [inline source/_dashboards/tile.markdown]:
```suggestion
  required: false
```
This cant be required as all types do not support it.

Comment (piitaya) [reply source/_dashboards/tile.markdown]: Yep, good catch 😅

File: source/_dashboards/tile.markdown [hunk 2]
```diff
@@ -71,3 +75,102 @@ Alternatively, the card can be configured using YAML:
...
+```yaml
+extras:
+  - type: "vacuum-commands"
+    commands:
+      - start_pause
+      - stop
+      - clean_spot
+      - locate
+      - return_home
+
```
Comment (ludeeus) [inline source/_dashboards/tile.markdown]:
```suggestion
```

File: source/_dashboards/tile.markdown [hunk 3]
```diff
@@ -71,3 +75,102 @@ Alternatively, the card can be configured using YAML:
 ## Available color tokens
 
 Some color tokens are available to colorize the tile card : `primary`, `accent`, `disabled`, `red`, `pink`, `purple`, `deep-purple`, `indigo`, `blue`, `light-blue`, `cyan`, `teal`, `green`, `light-green`, `lime`, `yellow`, `amber`, `orange`, `deep-orange`, `brown`, `grey`, `blue-grey`, `black` and `white`.
+
+## Tile extras
```
Comment (ludeeus) [inline source/_dashboards/tile.markdown]: We should probably have at least 1 full-example  of card configuration that includes the `extras` key, maybe use `vacuum-commands` as the example as that has more than just `type`

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**